site/public/js/user-profile.js

```diff
@@ -7,7 +7,14 @@ function showUpdatePrefNameForm() {
     $('[name="user_name_change"]', form).val('');
     $('#user-givenname-change').focus();
 }
-
+// eslint-disable-next-line no-unused-vars
+function showUpdatePronounsForm() {
+    $('.popup-form').css('display', 'none');
+    const form = $('#edit-pronouns-form');
+    form.css('display', 'block');
+    form.find('.form-body').scrollTop(0);
+    console.log('check');
```

Comment:
Should this line be deleted?

Reply:
I deleted the console.log line, but for the other lines, since that's what showUpdatePrefNameForm wrote, I think I should follow this format.